packages/charts/src/helper/data.ts

```
@@ -4,8 +4,10 @@ import {
   BORDER_DASHES,
   CHART_COLORS,
   PATTERN_SIZE,
+  OTHER_CHART_COLOR,
```

ESLint

[error] Member 'OTHER_CHART_COLOR' of the import declaration should be sorted alphabetically.

(sort-imports)